polkadot/node/subsystem-bench/src/lib/approval/message_generator.rs

```diff
@@ -560,12 +560,12 @@ struct TestSignInfo {
 	candidate_index: CandidateIndex,
 	/// The validator sending the assignments
 	validator_index: ValidatorIndex,
-	/// The assignments convering this candidate
+	/// The assignments covering this candidate
```

Comment:
Maybe a follow up PR: Periods at the end of a sentence :). I don't know why people don't do it, but it's all over the place.

Reply:
Yeah, I wouldn't mind making another PR for periods at the end of sentences and also sentences in comments without capital letters at the beginning.